lib/web/apiserver.go

```diff
@@ -1772,6 +1777,46 @@ func (h *Handler) changeUserAuthentication(w http.ResponseWriter, r *http.Reques
 	}, nil
 }
 
+func (h *Handler) trySettingConnectorNameToPasswordless(ctx *SessionContext, r *http.Request, req changeUserAuthenticationRequest) error {
+	if req.WebauthnCreationResponse == nil {
+		return nil
+	}
+
+	if !h.ClusterFeatures.GetCloud() {
+		return nil
+	}
+
+	authPreference, err := ctx.clt.GetAuthPreference(r.Context())
+
+	if err != nil {
+		return trace.Wrap(err)
```

Comment:
It could be quite common for some users to get an access denied when trying to get the auth preference, we should avoid logging an error in that case

Reply:
https://github.com/gravitational/teleport/pull/17834/commits/c3b2647b8e22eeb3515707383142a6bca2a8e964